app/jobs/submit_folio_request_job.rb

```diff
@@ -18,7 +18,7 @@ def perform(request_id, options = {})
 
     logger.debug("FOLIO response: #{response}")
     request.merge_ils_response_data(FolioResponse.new(response.with_indifferent_access))
-    request.save
+    request.save(validate: false) # By placing this request in the ILS, the item is no longer in a requestable state, so avoid validation.
```

Comment:
I can't think of any reason submitting the job should invalidate the request... so I guess this is probably fine? If that ever changes, I think there's a way we could instead relax the type validators for already-persisted requests but that seems... complicated.